app/assets/javascripts/notebook_actions.js.erb

```diff
@@ -592,22 +588,21 @@ $(document).ready(function(){
         cache: false,
         contentType: false,
         processData: false,
-        success: function(stagingResponse) {
+        success: function(stagingResponse){
           bootbox.hideAll();
           $('#editStagingID').val(stagingResponse['staging_id']);
           $('#editNotebookModal').modal('hide');
           $('#stageEdit').modal('show');
         },
-        error: function(response) {
+        error: function(response){
           bootbox.hideAll();
           var dirty = "dirty_words";
-          if(response.responseText.indexOf(dirty) >= 0){
+          if (response.responseText.indexOf(dirty) >= 0){
             dirtyWordCheck(response,'#editUploadErrorWarning','#editUploadSubmit');
           }
-          else{
-            $('#editStageErrorWarning').html('<strong>Error:</strong>' + response.responseText);
-            $('#editStageErrorWarning').attr('hidden',false);
-            $('#editStageSubmit').attr('disabled', false);
+          else {
+            $('#editUploadErrorWarning').html('<strong>Error: </strong>' + response.responseText);
```

Comment:
This looks like it was supposed to be a makeAlert because you removed the line that unhides the warning but left the text as the old way.